src/registrar/views/domain_request.py

```diff
@@ -834,6 +862,18 @@ def has_permission(self):
 
         return True
 
+    def get_context_data(self, **kwargs):
```

Comment:
Q: Why is this needed?

Reply:
This is too out of context for me to review, so I will withhold opinion on this code. If you NEED it reviewed, let's catch up on it tomorrow otherwise I will ignore it in my review (won't block)